js/main.js

```diff
@@ -9,6 +9,74 @@ var overAllScoreElement = document.querySelector('.overall-score');
 var xIcon = document.querySelector('.fa-xmark');
 var summaryButton = document.querySelector('#summary-button');
 var modalContainer = document.querySelector('.modal-container');
+var backToSearchButton = document.querySelector('.back-to-search-container').querySelector('button');
+var favSkipContainer = document.querySelector('.fav-skip-container');
+
+function removeStatsList() {
+  cityStatsCityStatsList.innerHTML = '';
+}
+
+function notificationPopUp() {
+  var notificationContainer = document.querySelector('.notification-container');
+  notificationContainer.className = 'notification-container';
+  setTimeout(hideNotificationPopUp, 750);
+}
+
+function hideNotificationPopUp() {
+  var notificationContainer = document.querySelector('.notification-container');
+  notificationContainer.className = 'notification-container notification-container-fade';
+}
+
+function populateNotification(favOrSkip) {
+  var notificationTextElement = document.querySelector('.notification-container').querySelector('p');
+  notificationTextElement.innerHTML = cityStatsCityNameElement.innerHTML + ' added to ' + favOrSkip + ' list';
+}
+
+var mobileFavSkipContainer = document.querySelector('.mobile-nav-container');
+
+function favSkipButtonClicked(event) {
+  var btnEventTarget = event.target.innerHTML;
+  populateNotification(btnEventTarget);
+  if (btnEventTarget === 'Favorite') {
+    addToFavorite();
+    notificationPopUp();
+  } else if (btnEventTarget === 'Skip') {
+    addToSkip();
+    notificationPopUp();
+  }
+}
+
+favSkipContainer.addEventListener('click', favSkipButtonClicked);
+
+function mobileFavSkipButtonClicked(event) {
+  if (event.target.innerHTML === 'Skip' || event.target.className === 'fa-regular fa-face-frown') {
```

Comment:
I would not base behavior on the text content of an element, you should apply a class to the element instead.